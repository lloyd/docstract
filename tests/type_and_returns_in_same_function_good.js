/** If so desired, the documentor may break up desc and
 *  type of return value (like in jsdoc) .*/

/**
 * @function
 * This function returns a String.
 * @return The name of the current user
 * @type String
 */
