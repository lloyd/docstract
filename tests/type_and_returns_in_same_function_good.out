{
  "desc": "If so desired, the documentor may break up desc and\n type of return value (like in jsdoc) .", 
  "filename": "tests/type_and_returns_in_same_function_good.js", 
  "functions": [
    {
      "desc": "This function returns a String.", 
      "name": "imDocumentedWithBothTypeAndReturn", 
      "returns": {
        "desc": "The name of the current user", 
        "type": "String"
      }
    }
  ], 
  "module": "type_and_returns_in_same_function_good"
}
